pkg/internal/test/gock.go

```diff
@@ -0,0 +1,31 @@
+package test
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/onsi/gomega"
+	"gopkg.in/h2non/gock.v1"
+)
+
+// EnsureGockRequestsHaveBeenMatched checks if all requests have been matched and all mocked requests have been used in the test
```

Comment:
it just checks if all requests have been matched, but not that all mocked requests have been used in the test - that was the function of `GetAll`

Reply:
Correct, the docs were for the original version with both checks